options/internal/include/mlibc/global-config.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef MLIBC_GLOBAL_CONFIG
+#define MLIBC_GLOBAL_CONFIG
+
+namespace mlibc {
+
+struct GlobalConfig {
+	GlobalConfig();
+	
+	bool debugMalloc;
+};
+
+inline const GlobalConfig &globalConfig() {
```

Comment:
I don't think that will work? does the static member get deduplicated despite being in each TU?

Reply:
Yeah, it works fine. The linker will just pick one:
```
$ nm --demangle libc.so | rg cached
0000000000530390 u guard variable for mlibc::globalConfig()::cached
000000000053038d u mlibc::globalConfig()::cached
```